src/dist/css/app.css

```diff
@@ -114,10 +114,34 @@ a.fa {
   position: relative;
 }
 
+.form-group-clear.col-auto .clear-input {
+  right: 0.7rem;
+  padding: 0.3rem 0.8rem;
+}
+
 .showInputClearLink {
   display: block;
 }
 
 .accordion-collapse {
   max-width: calc(100vw - var(--left-nav-offset) - 6rem);
 }
+
+.helix-logo {
+  background-image: url(../../images/helix.png);
+  background-position: center center;
+  background-repeat: no-repeat;
+  background-size: contain;
+  display: inline-block;
+  height: 35px;
+}
+
+#advSearchButton {
+  position: absolute;
```

Comment:
does having absolute here mess with the sizing of the page?

Reply:
@DarylCheung No, it stays contained/relative within the form element